components/builder-api-proxy/default.toml

```diff
@@ -30,6 +30,7 @@ provider       = "github"
 client_id      = ""
 authorize_url  = "https://github.com/login/oauth/authorize"
 redirect_url   = ""
+signup_url     = ""
```

Comment:
Should we default this to "https://github.com/join" ?

Reply:
Yes, good suggestion -- I didn't manage to get to it before it was merged. I'll submit a follow-up change for it.